metrics/prometheus.go

```diff
@@ -32,11 +32,19 @@ var (
 		Name:      "updated_count",
 		Namespace: "releases",
 	})
+
+	// TotalEvents is a metric for the number of events that have been handled by this operator
+	TotalEvents = prometheus.NewCounter(prometheus.CounterOpts{
+		Help:      "The number of events (create/delete/updates) processed by this operator",
+		Name:      "total_events",
```

Comment:
Total is normally at the end of a metric name, after the unit that is being counted.  It should probably be added to the other metrics names, if you dont mind while you're here.

Reply:
Done